sdks/python/tests/library_integration/openai/test_openai.py

```diff
@@ -58,6 +66,11 @@ def test_openai_client_chat_completions_create__happyflow(fake_streamer):
         opik.flush_tracker()
         mock_construct_online_streamer.assert_called_once()
 
+        # if "project name" was passed to tracker as None, default project name must be used,
+        # and it will be expected in results
+        if project_name is None:
+            project_name = OPIK_PROJECT_DEFAULT_NAME
```

Comment:
Minor: to avoid this kind of conditions in tests, so they're cleaner and more endurable, I actually advice to pass another parameter to the test: `project_name` and `expected_project_name`. So you can use the first one to parameterize the function calls and the second for assertions.